ws-nextjs-app/cypress/e2e/testsForAllPages.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable cypress/no-unnecessary-waiting */
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+import checkA11y from '../support/helpers/checkA11y';
```

Comment:
Is it possible to use the existing `checkA11y` script rather than a new one for the Next.js app?